### ISSUE TITLE
Misuses of cryptographic APIs

### ISSUE THREAD

**Issue by mhp0rtal** [zalando-stups/stups-spring-oauth2-support#68]
Hi
The following lines have cryptographic API misuses. <br> File name => stups-spring-oauth2-common/src/main/java/org/zalando/stups/spring/http/client/ClientHttpRequestFactorySelector.java: Line number => 142: API name => KeyStore:Operation on object of type java.security.KeyStore object not completed. Expected call to load<BR>

**Comment by jbellmann** [comment]
@mhp0rtal 
Indeed, after reading the docs in more detail I agree with you. On the other hand, in the line below you can see, that we simply trust all and never check any certificate chain.

I will forward this to our security team and will ask about their opinion.

Thanks for your support.
